object-manipulation-2/src/defaults.js

```diff
@@ -1 +1,8 @@
 /* exported defaults */
+function defaults(target, source) {
+  Object.keys(source).forEach(function (item) {
+    if (!Object.keys(target).includes(item)) {
```

Comment:
Here you just need to check if a property already exists on `target` or not. You can simplify this by just directly checking if the property is `undefined`.

For example:
```JAVASCRIPT
if(target[item] === undefined) {
   ...
}
```